nominal/core.py

```diff
@@ -72,6 +80,26 @@ def add_dataset(self, ref_name: str, dataset: Dataset | str) -> None:
         """
         self.add_datasets({ref_name: dataset})
 
+    def add_log_set(self, ref_name: str, log_set: LogSet | str) -> None:
+        """
+        Add a log set to this run.
+        """
+        self.add_log_sets({ref_name: log_set})
+
+    def add_log_sets(self, log_sets: Mapping[str, LogSet | str]) -> None:
```

Comment:
Separated into its own method rather than trying to combine with datasets

Reply:
should document what the key to the map is

Reply:
is adding multiple log sets a common workflow?